speech/audapter_runFrames.m

```diff
@@ -6,7 +6,11 @@
 %   returned by Audapter).
 
 % set Audapter param fields
-gender = 'female'; % dummy var for getting default params; overwritten by P if provided
+if ~isempty(p.gender)
+    gender = p.gender;
+else
+    gender = 'female'; % default gender is female
+end
```

Comment:
Whatever struct you're trying to pull gender from isn't available when you're checking it. `p` was just a 1x1 double with a value equal to my target LPC order when I ran through a workflow.

Workflow: Run `run_vsaAdapt2_expt([],1)`. Get to checkLPC window. Change LPC order. Error occurs.

MATLAB error msg:

> Running Audapter with nLPC = 16...
> Dot indexing is not supported for variables of this type.
> Error in audapter_runFrames (line 9)
> if ~isempty(p.gender)
> Error in check_audapterLPC>changeLPC (line 186)
>             data(d) = audapter_runFrames(UserData.data(d),UserData.nLPC); 
> Error using run_vsaAdapt2_expt (line 148)
> Error while evaluating UIControl Callback.

Reply:
Can you do a pull of current-studies and try again? I forgot to mention that we made a change to check_audapterLPC to accommodate for this runFrames stuff, but I didn't include it in this pull request as it's in that other repo. My apologies for the confusion!

Reply:
This was resolved by me pulling current-studies. Lesson learned...